.github/CODEOWNERS

```diff
@@ -12,22 +12,22 @@
 /sdk/boms/azure-sdk-bom/                            @alzimmermsft @jonathangiles @srnagar @hemanttanwar @anuchandy
 
 # Service teams
-/sdk/appconfiguration/                              @mssfang @alzimmermsft @conniey @sima-zhu
-/sdk/batch/                                         @xingwu1 @bgklein @matthchr
-/sdk/core/                                          @alzimmermsft @jianghaolu @srnagar @hemanttanwar @anuchandy
-/sdk/core/azure-core-tracing-opentelemetry/         @samvaity @alzimmermsft
-/sdk/cosmos/                                        @moderakh @kushagraThapar @David-Noble-at-work @kirankumarkolli @mbhaskar
+/sdk/appconfiguration/                              @mssfang @alzimmermsft @conniey @sima-zhu                     %AzConfig
+/sdk/batch/                                         @xingwu1 @bgklein @matthchr                                   %Batch
+/sdk/core/                                          @alzimmermsft @jianghaolu @srnagar @hemanttanwar @anuchandy   %Azure.Core
+/sdk/core/azure-core-tracing-opentelemetry/         @samvaity @alzimmermsft 
+/sdk/cosmos/                                        @moderakh @kushagraThapar @David-Noble-at-work @kirankumarkolli @mbhaskar   %Cosmos
 /sdk/cosmos/azure-spring-data-cosmosdb/             @kushagraThapar @saragluna @yiliuTo @chenrujun @zhoufenqin @jialindai
```

Comment:
@jialindai can you confirm that this should be auto-labeled with https://github.com/Azure/azure-sdk-for-java/labels/azure-spring-data-cosmos ?

Reply:
@joshfree `azure-spring-data-cosmos` is the label I created for triaging spring issues when I copied them over from spring-data-cosmosdb repo. 
I think Jialin created a different label - `azure-spring.cosmos`

I think we should just merge these two labels to avoid confusion.

Reply:
Thanks @joshfree @kushagraThapar